dev/io.openliberty.xmlBinding.3.0.internal.tools/jaxb_xjc.bnd

```diff
@@ -16,8 +16,8 @@ Bundle-Name: IBM JAXB XJC Tool
 
 Require-Compiler: true
 
-Require-Bundle: io.openliberty.jaxb.3.0.internal.tools; version="[1,1.0.100)", \
- io.openliberty.jakarta.jaxb.3.0; version="[1,1.0.100)"; location="dev/api/spec/", \ \
+Require-Bundle: io.openliberty.xmlBinding.3.0.internal.tools; version="[1,1.0.100)", \
+ io.openliberty.jakarta.xmlBinding.3.0; version="[1,1.0.100)"; location="dev/api/spec/", \ \
  com.ibm.ws.logging; version="[1,1.0.100)"
  
 Command-Class: io.openliberty.jaxb.tools.XJC
```

Comment:
Should the package name change to be xmlbinding instead of jaxb?

Reply:
Should the nls files paths be updated to remove reference to jaxb in the package?  Also the nls files should be updated to remove reference to JAXB.

Reply:
Sure

Reply:
As pointed out in another review comment the nls files didn't move to a new location.